src/main/java/io/moderne/organizations/OrganizationStructureService.java

```diff
@@ -167,16 +73,41 @@ void logStructure(Map<String, OrganizationRepositories> organizations) {
         printTree(tree, null, 0);
     }
 
-    public static void printTree(Map<String, List<OrganizationRepositories>> tree, String parentName, int level) {
+    public static void printTree(Map<String, List<OrganizationRepositories>> tree, @Nullable String parentName, int level) {
         for (OrganizationRepositories org : tree.getOrDefault(parentName, Collections.emptyList())) {
             String indent = "\t".repeat(level);
             log.debug("{}{} [{}]", indent, org.name(), org.repositories().size());
             printTree(tree, org.name(), level + 1);
         }
     }
 
-    RepositoryInput determineRepository(String cloneUrl, String branch) {
-        GitRemote gitRemote = gitRemoteParser.parse(cloneUrl);
-        return new RepositoryInput(gitRemote.getPath(), gitRemote.getOrigin(), branch);
+    private void determineRepositoriesForOrganization(
+            Organization<Object> organization,
+            Map<String, OrganizationRepositories> organizations,
+            @Nullable String parentId
+        ) {
+        organizations.put(organization.getName(), new OrganizationRepositories(
+                organization.getId(),
+                organization.getName(),
+                mapRepositories(organization.getRepositories()),
+                List.of(CommitOption.values()),
+                requireNonNull(organization.getParent()).isRoot() ? null : organization.getParent().getId()
+        ));
```

Comment:
removed the loading and setting of the `CommanOptions` file and default the value to all (`List.of(CommitOption.values())`)